dev/com.ibm.ws.app.manager/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -48,6 +48,7 @@
 		<AD id="useJandex" name="%appmgmt.useJandex" description="%appmgmt.useJandex.desc" required="false" type="Boolean"/>
 		<AD id="startTimeout" name="%appmgmt.startTimeout" description="%appmgmt.startTimeout.desc" required="false" type="String" default="30s" ibm:type="duration(s)" />
 		<AD id="stopTimeout" name="%appmgmt.stopTimeout" description="%appmgmt.stopTimeout.desc" required="false" type="String" default="30s" ibm:type="duration(s)" />
+		<AD id="targetDir" name="%appmgmt.targetDir" description="%appmgmt.targetDir.desc" required="false" type="String" />
```

Comment:
Use a default value here (equal to the current constant that we use for the expanded apps dir) so that you can get rid of all the null checking

Reply:
Added this as default="${server.config.dir}/apps/expanded/"